packages/account/src/Sections/Profile/PhoneVerification/ConfirmPhoneNumber.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PhoneVerificationCard from './PhoneVerificationCard';
+import { Button, Text } from '@deriv-com/quill-ui';
+import { Localize, localize } from '@deriv/translations';
+import { Input } from '@deriv/components';
+import { useSettings } from '@deriv/api';
+
+const ConfirmPhoneNumber = () => {
+    const { data: accountSettings } = useSettings();
+
+    const phoneNumber = accountSettings.phone || '';
+    return (
+        <PhoneVerificationCard>
+            <Text bold>
+                <Localize i18n_default_text='Confirm your phone number' />
+            </Text>
+            <Input label={localize('Phone number')} disabled value={phoneNumber} />
```

Comment:
As per design , this field should be enabled. isnt?

Reply:
will remove this disable property